packages/expo-cellular/src/Cellular.types.ts

```diff
@@ -20,4 +20,114 @@ export enum CellularGeneration {
+   * ```
+   */
+  allowsVoip: boolean | null;
+
+  // @needsAudit
+  /**
+   * The ISO country code for the user’s cellular service provider. On iOS, the value is `null` if any
+   * of the following apply:
+   * - The device is in airplane mode.
+   * - There is no SIM card in the device.
+   * - The device is outside of cellular service range.
+   *
+   * On web, this returns `null`.
+   *
+   * # Examples
+   * ```ts
+   * Cellular.isoCountryCode; // "us" or "au"
+   * ```
+   */
+
+  isoCountryCode: string | null;
+  // @needsAudit
+  /**
+   * The name of the user’s home cellular service provider. If the device has dual SIM cards, only the
+   * carrier for the currently active SIM card will be returned. On Android, this value is only
+   * available when the SIM state is [`SIM_STATE_READY`](https://developer.android.com/reference/android/telephony/TelephonyManager.html#SIM_STATE_READY).
+   * Otherwise, this returns `null`.
+   *
+   * On iOS, if you configure a device for a carrier and then remove the SIM card, this property
+   * retains the name of the carrier. If you then install a new SIM card, its carrier name replaces
+   * the previous value of this property. The value for this property is `null` if the user never
+   * configured a carrier for the device.
+   *
+   * On web, this returns `null`.
+   *
+   * # Examples
+   * ```ts
+   * Cellular.carrier; // "T-Mobile" or "Verizon"
+   * ```
+   */
+  carrier: string | null;
+
+  // @needsAudit
+  /**
+   * The mobile country code (MCC) for the user’s current registered cellular service provider.
+   * On Android, this value is only available when SIM state is [`SIM_STATE_READY`](https://developer.android.com/reference/android/telephony/TelephonyManager.html#SIM_STATE_READY). Otherwise, this
+   * returns `null`. On iOS, the value may be null on hardware prior to iPhone 4S when in airplane mode.
+   * Furthermore, the value for this property is `null` if any of the following apply:
+   * - There is no SIM card in the device.
+   * - The device is outside of cellular service range.
+   *
+   * On web, this returns `null`.
+   *
+   * # Examples
+   * ```ts
+   * Cellular.mobileCountryCode; // "310"
+   * ```
+   */
+  mobileCountryCode: string | null;
+
+  // @needsAudit
+  /**
+   * The ISO country code for the user’s cellular service provider. On iOS, the value is `null` if
+   * any of the following apply:
+   * - The device is in airplane mode.
+   * - There is no SIM card in the device.
+   * - The device is outside of cellular service range.
+   *
+   * On web, this returns `null`.
+   *
+   * # Examples
+   * ```ts
+   * Cellular.mobileNetworkCode; // "260"
+   * ```
+   */
+  mobileNetworkCode: string | null;
+
+  // @needsAudit
+  /**
+   * Describes the current generation of the cellular connection. It is an enum with these possible
+   * values:
+   * - UNKNOWN
+   * - CELLULAR_2G
+   * - CELLULAR_3G
+   * - CELLULAR_4G
+   * - CELLULAR_5G
+   */
```

Comment:
There is no need to list enum values in the field description. The type name should by linked to the section where those values are listed and explained.